### DISCUSSION TITLE
Where to report documentation/help errors?

### DISCUSSION THREAD

**Discussion by Who3D** [WinMerge/winmerge#1003] [Q&A]
Hi all, I'm new to WinMerge so please be gentle with me.

Documentation errors are not strictly bugs, so I'm wondering how best to report them in the hopes of getting them fixed?

For example, I feel that from the "Select Files or Folders dialog" section:

> 4.1. Using project files
> To use a project file In the WinMerge window, click File → Open Project. WinMerge reads the paths from the project file. If the project file has two or three paths, **two respective three paths** are opened. If only one path is given or **one of paths is invalid**, WinMerge opens the Select Files or Folders dialog, where you can add or correct the path information. Here are a few things to consider:

Should probably be something more like

> 4.1. Using project files
> To use a project file In the WinMerge window, click File → Open Project. WinMerge reads the paths from the project file. If the project file has two or three paths, **two or three paths** are opened. If only one path is given or **one of the paths is invalid**, WinMerge opens the Select Files or Folders dialog, where you can add or correct the path information. Here are a few things to consider:

I appreciate this is a rather tiny issue, and beg your indulgence :)

Cheers,

Cliff

PS I am terrible at having typos in my own text (or excellent I guess, depending on your perspective) so no slight is intended yo whoever wrote the above in the documentation. If I find anything else to amend I am sure to make plenty of mistakes in my posts!

**Comment by sdottaka** [comment]
Fixed with commit 6464a07 

Thank you for the report.
